apps/frontend/tsconfig.json

```diff
@@ -20,12 +20,14 @@
     "noImplicitOverride": true,
     "noPropertyAccessFromIndexSignature": true,
     "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "strictPropertyInitialization": false
```

Comment:
It might be cumbersome in the beginning, but I would recommend to not disable strict flags in the `tsconfig`. In the long run you want your tools to be as strict as possible, because this reduces the potential for errors on your side.